bcda/cclf/cclf.go

```diff
@@ -417,53 +405,74 @@ func sortCCLFArchives(cclfMap *map[string]map[int][]*cclfFileMetadata, skipped *
 			return nil
 		}
 
-		zipReader, err := zip.OpenReader(path)
+		zipReader, err := zip.OpenReader(filepath.Clean(path))
+		defer zipReader.Close()
+
 		if err != nil {
+			// this skipped count is not entirely accurate.
```

Comment:
Why?   We should only count skips for the top-level zips; not the individual files within the zips.  That would get very noisy, since we automatically skip 10 files in each archive.

Reply:
yup u right, changed :)